### ISSUE TITLE
Licence Issue

### ISSUE THREAD

**Issue by chrismacp** [moll/node-mitm#65]
Hi,

This package is being flagged in Snyk as a vulnerability due to the licence. I found a [similar issue](https://github.com/winkjs/wink-bm25-text-search/issues/14) in another package which suggested it was due to deprecated licence. I'm not an expert on this so can't confirm that is correct but if so it would be great to update this.

Thanks

**Comment by moll** [comment]
Thanks for the heads up! I'll investigate and possibly write to them.

**Comment by moll** [comment]
Do you happen to have some output from Snyk's tools that highlight Mitm.js, @chrismacp? I'm trying to reproduce this via Snyk's web interface, but so far haven't been able to.

**Comment by moll** [comment]
When I look at their database, it doesn't seem to mark it as vulnerable, either: https://snyk.io/vuln/npm:mitm. Could this be a bug with their CLI app?

**Comment by chrismacp** [comment]
Hi, 
Thanks for the super fast response! I found out a little more and I think it's actually not Snyk, but my company that has defined this licence type as a "high" vulnerability within our Snyk settings. Sorry that wasn't immediately clear to me when I reported this issue. I've just been reading through all the licence related docs I could find. Similar to [Google's banned licence list](https://opensource.google/docs/thirdparty/licenses/#banned), my company doesn't permit the use of anything with this licence unfortunately. Will have to find an alternative.

Thanks again though :)
